rendertime/rendertime.php

```diff
@@ -53,8 +53,11 @@ function rendertime_page_end(App $a, string &$o)
 
 	$duration = microtime(true) - $profiler->get('start');
 
-	$ignored_modules = ['fbrowser'];
-	$ignored = in_array(DI::args()->getModuleName(), $ignored_modules);
+	$ignored_modules = [
+		\Friendica\Module\Profile\Photos\Browser::class,
+		\Friendica\Module\Profile\Attachment\Browser::class,
+	];
+	$ignored = in_array(DI::router()->getModuleClass(), $ignored_modules);
```

Comment:
Doesn't this runs the router again?

Reply:
yep, you're right ...

Reply:
Now it's cached :)
See https://github.com/friendica/friendica/pull/12277/commits/4e53ba0c20aef86d7a5ef202faed498b4181bb76